packages/network/test/integration/proxy-connections.test.ts

```diff
@@ -68,39 +73,36 @@ describe('Proxy connection tests', () => {
             tracker?.stop(),
             onewayNode?.stop(),
             contactNode?.stop(),
-            contactNode2?.stop(),
+            contactNode2?.stop()
         ])
     })
 
     it('publisher node can form proxy connections', async () => {
-        await onewayNode.openProxyConnection(defaultStreamPartId, 'contact-node', ProxyDirection.PUBLISH, 'publisher')
```

Comment:
All operations in this test are using this stream part. Maybe it would make sense to drop the `default` prefix because it is a bit misleading.